main.go

```diff
@@ -94,8 +103,16 @@ func main() {
 		dief("looking for test data: %v\n", err)
 	}
 	for _, name := range remove {
-		if err := os.RemoveAll(name); err != nil {
-			dief("removing %v: %v\n", name, err)
+		err := os.RemoveAll(name);
+		if err != nil {
+			failedToRemove[name] = err
+		}
+	}
+	
+	if len(failedToRemove) > 0 {
+		fmt.Fprintf(os.Stderr, "\n# Failed to remove below item, please remove manually\n")
+		for name, err := range failedToRemove {
```

Comment:
good to know that there is solution. let me give it try to see if solve the issue. 

it is debatable i think whether to failed the whole process. 
1) should rarely happen
2) in my case, only ".git" folder was failed to be deleted. if we let the operation continue, manual delete one folder i am still good to go.  

so what do you think to have this backup handling?